### PR TITLE
AAP-2019 - Note and new topic for removing PVCs added

### PR DESCRIPTION
**Pull request by dcdacosta**
Fixes issue https://issues.redhat.com/browse/AAP-2019

Changes proposed in this pull request:

- Added note that PVCs are not automatically deleted along with the app.
- Included new procedure for Finding and deleting PVCs

Preview link (VPN required): http://file.rdu.redhat.com/~ddacosta/aap2019/master.html

### CODE REVIEW

File: downstream/titles/aap-operator-installation/master.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <meta charset="UTF-8">
```
Comment (oraNod) [inline downstream/titles/aap-operator-installation/master.html:4]: @dcdacosta Hi, I know this PR is currently a WIP but I noticed the `master.html` file is included here. There is a `.gitignore` file in the repo that should keep it from getting picked up. I'm not sure why that isn't working but you might want to delete this so it doesn't get added to the repo. Cheers.

Comment (dcdacosta) [reply downstream/titles/aap-operator-installation/master.html]: Done!

**Review by rooftopcellist — APPROVED** [review]
@dcdacosta the docs changes look good to me.  I don't know much about how this repo is set up, but I just wanted to make sure that the deletion of `downstream/titles/aap-operator-installation/master.html` was intentional.  If so, then let's go ahead and merge this!

**Review by oraNod — APPROVED** [review]
LGTM

**Comment by oraNod** [comment]
Thanks @dcdacosta Can you create the backport PRs please?

**Pull request merged**